stuff/pipelines/test_producer_contest.py

```diff
@@ -25,9 +26,9 @@
 import lib.db   # pylint: disable=wrong-import-position
 
 @dataclasses.dataclass
-class Message:
+class MessageSavingCallback:
     '''class to save message'''
-    message: Dict[str, Any] = dataclasses.field(default_factory=dict)
+    message: Optional[contest_callback.ContestCertificate] = dataclasses.field(default_factory=dict)
```

Comment:
```suggestion
    message: Optional[contest_callback.ContestCertificate] = None
```

esto debería estar no-establecido por default para darnos cuenta que no se mandó llamar.

Reply:
Listo.